### PR TITLE
Document compression configuration for server log

### PR DESCRIPTION
**Pull request by willshen**
## Description
Document compression configuration for server log

## Release notes

(x) This is not user-visible or docs only and no release notes are required.
( ) Release notes are required, please propose a release note for me.
( ) Release notes are required, with the following suggested text:

```markdown
# Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

File: docs/src/main/sphinx/admin/properties-logging.rst [hunk 1]
```diff
@@ -56,6 +56,15 @@ rotation replaces old content.
 
 The maximum file size for the general application log file.
 
+``log.compression``
+^^^^^^^^^^^^
```
Comment (electrum) [inline docs/src/main/sphinx/admin/properties-logging.rst:60]: This needs to match the length of the text above

**Review by electrum — APPROVED** [review]

**Comment by mosabua** [comment]
Refer to the readme file in the docs folder to see how you can build the docs locally to verify.

https://github.com/trinodb/trino/tree/master/docs

File: docs/src/main/sphinx/admin/properties-logging.rst (hunk 1 above)
Comment (willshen) [reply docs/src/main/sphinx/admin/properties-logging.rst]: Thank you for catching this - this is fixed now.

**Comment by willshen** [comment]
@mosabua Thank you for the pointer - I ran the build to make sure it built successfully after the fix now.

**Comment by mosabua** [comment]
Thank you for your contribution @willshen !

**Pull request merged**